app/tasks/maintenance_tasks/task.rb

```diff
@@ -27,7 +39,7 @@ def named(name)
       # @return [Array<Class>] the list of classes.
       def available_tasks
         load_constants
-        descendants
+        descendants.reject(&:abstract_class?)
```

Comment:
I think we could do without `abstract_class?` by doing this:
```suggestion
        descendants.without(CsvTask)
```

Reply:
I's suspecting that using a superclass might be the best option for implementing custom iteration Tasks (see https://github.com/Shopify/maintenance_tasks/pull/310#issuecomment-766843596), which prompted me to keep `abstract_class?` in here. But we can always go with the simplest option for now, and only implement `abstract_class`  once we actually have another class that needs to be abstract 😁